Libraries/Text/Text.js

```diff
@@ -153,6 +153,16 @@ const Text: React.AbstractComponent<
       : processColor(restProps.selectionColor);
 
   let style = restProps.style;
+
+  let _selectable = restProps.selectable;
+  if (style && style.userSelect !== undefined) {
```

Comment:
Style needs to be flattened as it could be an array

Reply:
@necolas i have pushed the changes